src/build.py

```diff
@@ -1325,57 +1325,61 @@ def Test(self):
     self.runnable()
 
 
+def GetTortureDir(name, opt):
+  dirs = {
```

Comment:
This is cleverly done, it makes the special cases explicit while cutting down on caller boilerplate and covering all the cases. Nice